cruise-control-core/src/main/java/com/linkedin/cruisecontrol/servlet/handler/Request.java

```diff
@@ -19,11 +16,9 @@
 public interface Request extends CruiseControlConfigurable {
 
   /**
+   * @param handler The request handler
    * Handle the request and populate the response.
-   *
-   * @param request Http servlet request.
-   * @param response Http servlet response.
    */
-  void handle(HttpServletRequest request, HttpServletResponse response)
-      throws IOException, ExecutionException, InterruptedException;
+  void handle(CruiseControlRequestContext handler)
```

Comment:
Is this interface change backwards compatibility? Or you simply use a CruiseControlRequestContext wrapper to replace the old request/response params everywhere?

Reply:
We use CruiseControlRequestContext and replaced the request/response params everywhere.